### PR TITLE
Recipe app  project

### PR DESCRIPTION
**Pull request by sediqullahbadakhsh**
In this PR we @andres-condezo  and @sediqullahbadakhsh created: 

- Set up the repository on GitHub and use Gitflow.
- Set up Devise for authentication.
- Set up RSpec and Capybara testing libraries.
- followed the layout of the wireframes provided
- Login page and registration page build with Devise

- Food list

1. displays a list of food added by a logged-in user with the quantity of a given food
2. leads to a form that allows users to add new food.

- Recipes List

1. Displays a list of recipes created by the logged-in user as in the wireframe
2. leads to recipe details
3. if the user is the owner of the recipe, able to delete it. 

- Public recipe list:

1. displays a list of all public recipes ordered by newest as in the wireframe.

- Recipe details:

1. If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
2. If the user is the owner of the recipe, should lead to the form that allows the user to add new food.

- General shopping list view

1. shows the list of food that is missing for all recipes of the logged-in user
2. counts the total food items and total price of the missing food

- Create a navigation menu that allows users to open all of the pages.

Thanks for your time.

### CODE REVIEW

File: spec/system/login_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    it 'should have login text' do
+      expect(page).to have_content 'Log in'
+    end
+    it 'should have input email' do
+      expect(page).to have_content 'Email'
+    end
+    it 'should have password input' do
+      expect(page).to have_content 'Password'
+    end
+    it 'should have login button' do
```
Comment (rotimiazeez) [inline spec/system/login_spec.rb:19]: - Kindly make sure the login test is passing
Screenshot from 2022-07-29 20-40-58

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM 👏🏽,
## Great job you did here 💯 
- Good UI 😃 
- Great teamwork/peer code review 👍🏽 
- Good README 🥇 
- Descriptive PR 💯 
- Good commit messages 🏷️
- Correct git-flow followed 👍🏽 
- No linter errors ✔️
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/system/login_spec.rb (hunk 1 above)
Comment (sediqullahbadakhsh) [reply spec/system/login_spec.rb:19]: Thanks for your kind review, we made some changes after the test and we forgot to check the test and run the tests again, and after requesting for Review we noticed that there was one test failing, although we did fix it I guess we were too late.

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @sediqullahbadakhsh and @andres-condezo 😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**